lib/slave/include/kickcat/slave/Slave.h

```diff
@@ -0,0 +1,36 @@
+#ifndef SLAVE_STACK_INCLUDE_SLAVE_H_
+#define SLAVE_STACK_INCLUDE_SLAVE_H_
+
+
+#include "kickcat/PDO.h"
+#include "kickcat/AbstractESC.h"
+#include "kickcat/Mailbox.h"
+#include "kickcat/ESMStates.h"
+
+namespace kickcat::slave
+{
+    class Slave final
+    {
+    public:
+        Slave(AbstractESC* esc, PDO* pdo);
+
+        void set_mailbox(mailbox::response::Mailbox* mbx);
+        void start();
+        void routine();
+        State getState();
+        void setOutputDataValid(bool isValid);
```

Comment:
Not sure we will keep this API in the future.
It makes the Application responsible for the check.
The application could provide a callback responsible for checking the output validity which would be used by safeOpState. (As well as default valid output to be exposed as long as we are in safeOp)